numpy/core/tests/test_multiarray.py

```diff
@@ -194,6 +194,17 @@ def test_fill(self):
             y[...] = 1
             assert_equal(x, y)
 
+    def test_fill_max_uint64(self):
+        x = empty((3, 2, 1), dtype=uint64)
+        y = empty((3, 2, 1), dtype=uint64)
+        value = 2**64 - 1
+        y[...] = value
+        try:
+            x.fill(value)
+        except:
+            raise RuntimeError(getexception())
```

Comment:
I wasn't sure if for some reason it was standard to use RuntimeException or not. I based this on another test up above. I'll pull it out.